### ISSUE TITLE
Dockerfile should use fix-width font

### ISSUE THREAD

**Issue by jakirkham** [docker/hub-beta-feedback#177]
It is hard to follow code files that don't use fixed-width font. Would be nice if they were switched to fixed-width font.

**Comment by ChristopherBiscardi** [comment]
Indeed it should. This is PR'd and waiting to be merged.

**Comment by jakirkham** [comment]
Thanks for sharing. Looking forward to it.

**Comment by jakirkham** [comment]
...or is this already done?

**Comment by jakirkham** [comment]
This looks fixed to me. Closing...
